packages/@react-stately/calendar/package.json

```diff
@@ -19,6 +19,7 @@
   "dependencies": {
     "@babel/runtime": "^7.6.2",
     "@internationalized/date": "3.0.0-alpha.4",
+    "@react-aria/i18n": "^3.3.8",
```

Comment:
seems erroneous

Reply:
yup, think I had some accidental left over changes in my editor. Removed by https://github.com/adobe/react-spectrum/pull/2850/commits/b319c25dd80b5e150511b4a6e4e894edf684e96e